doc/release.rst

```diff
@@ -4,6 +4,187 @@ Release Notes
+-  Several plots produced by ``plot`` class methods now have default titles (`#453 <https://github.com/giotto-ai/giotto-tda/pull/453>`_).
+-  Infinite deaths are now plotted by ``plot_diagrams`` (`#461 <https://github.com/giotto-ai/giotto-tda/pull/461>`_).
+-  Possible multiplicities of persistence pairs in persistence diagram plots are now indicated in the hovertext (`#454 <https://github.com/giotto-ai/giotto-tda/pull/454>`_).
+-  ``plot_heatmap`` now accepts boolean array input (`#444 <https://github.com/giotto-ai/giotto-tda/pull/444>`_).
+
+New tutorials and examples
+--------------------------
+
+The following new tutorials have been added:
+
+-  `Topology of time series <https://giotto-ai.github.io/gtda-docs/0.3.0/notebooks/time_series_classification.html>`_, which explains the theory of the Takens time-delay embedding and its use with persistent homology, demonstrates the new ``API`` of several components in ``gtda.time_series``, and shows how to construct time series *classification* pipelines in ``giotto-tda`` by partially reproducing `arXiv:1910:08245 <https://arxiv.org/abs/1910.08245>`_.
+-  `Topology in time series forecasting <https://giotto-ai.github.io/gtda-docs/0.3.0/notebooks/time_series_forecasting.html>`_, which explains how to set up time series *forecasting* pipelines in ``giotto-tda`` via ``TransformerResamplerMixin``s and the ``giotto-tda`` ``Pipeline`` class.
+-  `Topological feature extraction from graphs <https://giotto-ai.github.io/gtda-docs/0.3.0/notebooks/persistent_homology_graphs.html>`_, which explains what the features extracted from directed or undirected graphs by ``VietorisRipsPersistence``, ``SparseRipsPersistence`` and ``FlagserPersistence`` are.
+-  `Classifying handwritten digits <https://giotto-ai.github.io/gtda-docs/0.3.0/notebooks/MNIST_classification.html>`_, which presents a fully-fledged machine learning pipeline in which cubical persistent homology is applied to the classification of handwritten images from he MNIST dataset, partially reproducing `arXiv:1910.08345 <https://arxiv.org/abs/1910.08345>`_.
+
+Utils
+-----
+
+-  A ``check_collection`` input validation function has been added (`#491 <https://github.com/giotto-ai/giotto-tda/pull/491>`_).
+-  ``validate_params`` now accepts ``"in"`` and ``"of"`` keys simultaneously in the ``references`` dictionaries, with ``"in"`` used for non-list-like types and ``"of"`` otherwise (`#502 <https://github.com/giotto-ai/giotto-tda/pull/502>`_).
+
+Installation improvements
+-------------------------
+
+-  ``pybind11`` is now treated as a standard git submodule in the developer installation (`#459 <https://github.com/giotto-ai/giotto-tda/pull/459>`_).
+-  `pandas <https://github.com/giotto-ai/giotto-tda/pull/508>`_ is now part of the testing requirements when intalling from source (`#508 <https://github.com/giotto-ai/giotto-tda/pull/508>`_).
```

Comment:
This does not render properly in the draft of the release: https://github.com/giotto-ai/giotto-tda/releases

Reply:
It should not have come with a hyperlink. I fixed this in both places.